.github/infra/cloudbuild-configure-triggers.yaml

```diff
@@ -17,18 +17,12 @@
 
 steps:
 - name: 'google/cloud-sdk:slim'
-  id: Configure webui unit tests
-  waitFor: ['-']
+  id: Configure deployment
```

Comment:
nit: this sounds a bit generic. Maybe "Configure build triggers" or "Deploy build triggers"?

Reply:
Renamed